webpack/components/SCCProductPage/components/common/SCCSubscribedProductsExpander/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import SCCGenericExpander from '../SCCGenericExpander';
+
+const SCCSubscribedProductsExpander = ({ setExpandAllInParent }) => {
+  const selectOptionSubscribedOnly = __('Show only subscribed products');
+  const selectOptionShowAll = __('Show all products');
+  const initSelectionOption = __('Show/Hide unsubscribed');
+
+  return (
+    <>
+      <SCCGenericExpander
+        setInParent={setExpandAllInParent}
+        selectOptionOpen={selectOptionShowAll}
+        selectOptionClose={selectOptionSubscribedOnly}
+        initLabel={initSelectionOption}
+      />
+    </>
+  );
+};
```

Comment:
just something that I noticed in several components,
if there's only one child inside the React.Fragment, a.k.a `<></>` 
then you don't need the Fragment :)

also if the translations are used only once, you could pass them straight on the prop line, so this component could look like (didn't test):

```suggestion
const SCCSubscribedProductsExpander = ({ setExpandAllInParent }) => (
      <SCCGenericExpander
        setInParent={setExpandAllInParent}
        selectOptionOpen={__('Show all products')}
        selectOptionClose={__('Show only subscribed products')}
        initLabel={__('Show/Hide unsubscribed')}
      />
);
```

Reply:
That would be much more concise!